sos/report/__init__.py

```diff
@@ -355,6 +359,25 @@ def add_parser_options(cls, parser):
         preset_grp.add_argument("--del-preset", type=str, action="store",
                                 help="Delete the named command line preset")
 
+        # Group the proxy options together
+        proxy_grp = parser.add_argument_group(
+            'HTTP/HTTPS Proxy Options',
+            'These options control the configuration of a '
+            'proxy server to be used for uploads.'
+        )
+        proxy_grp.add_argument('--proxy-user',
+                               default=None,
+                               help='Specify the User for the proxy server')
+        proxy_grp.add_argument('--proxy-pass',
+                               default=None,
+                               help='Specify the Password for the proxy user')
+        proxy_grp.add_argument('--proxy-host',
+                               default=None,
+                               help='Specify the host for the proxy server')
+        proxy_grp.add_argument('--proxy-port',
+                               default=None,
+                               help='Specify the Port for the proxy server')
+
```

Comment:
We need to enhance man pages accordingly.

When you will describe `--proxy-pass`, it is worth mentioning that using the option on cmdline reveals the password to anybody running `ps` at the same time.